core/src/main/scala/org/apache/spark/sql/CachedDataFrame.scala

```diff
@@ -216,11 +216,13 @@ class CachedDataFrame(snappySession: SnappySession, queryExecution: QueryExecuti
     var pool = snappySession.sessionState.conf.activeSchedulerPool
     // Check if it is pruned query, execute it automatically on the low latency pool
     if (isLowLatencyQuery && pool == "default") {
-      if (snappySession.sparkContext.getPoolForName(Constant.LOW_LATENCY_POOL).isDefined) {
-        pool = Constant.LOW_LATENCY_POOL
+        if (snappySession.sparkContext.getPoolForName(Constant.LOW_LATENCY_POOL).isDefined) {
```

Comment:
Does this cause all the query in SnappySink also to go in streaming pool?

Reply:
This is done to avoid overwriting `streaming` pool from this code. We are setting streaming pool when the streaming thread started. If any of the queries inside the sink will execute this code, then `streaming` pool was getting overwritten.

Reply:
this particular change is not required and hence reverted.